### ISSUE TITLE
SyntaxError: JSON.parse causes TestCafé test to fail

### ISSUE THREAD

**Issue by mansn** [DevExpress/testcafe-hammerhead#2163]
### What is your Scenario?
A TestCafé test fails due to `Uncaught SyntaxError: JSON.parse`. I cannot reproduce the error manually (without TestCafé). I reproduced the error using the Hammerhead playground. This only seems to occur when the tested website makes a HTTP GET Request which gets following response body:
`errordata=Game+Server+Error+Trace%3A+%0AFailed+to+send+events+to+Kafka&error=TECHNICAL_ERROR&errorcode=0`
I've provided a link to a web page (below) which basically just contains that text. You can reproduce the error by opening that url with the Hammerhead playground.
 
### What is the Current behavior?

`Uncaught SyntaxError: JSON.parse` occurs.

### What is the Expected behavior?

`Uncaught SyntaxError: JSON.parse` doesn't occur.
 
### What is your public web site URL?

https://csb-n2m2l.netlify.com/

Your website URL (or attach your complete example): 
https://codesandbox.io/embed/crazy-euclid-n2m2l
<details>
<summary>Your complete app code (or attach your test files):</summary>

 
```js
 
```
</details>

<details>
<summary>Screenshots:</summary>

```

```
</details>

### Steps to Reproduce:

1. Using the Hammerhead playground, open (proxy) the following url: https://csb-n2m2l.netlify.com/
2. See the error: 
```Uncaught SyntaxError: JSON.parse
        at exports.parse (hammerhead.js:8840)
        at Function._getMessageData (hammerhead.js:13406)
        at MessageSandbox._onMessage (hammerhead.js:13413)
        at fastApply (hammerhead.js:24225)
        at onMessageHandler (hammerhead.js:13492)
        at hammerhead.js:13007
```
 
### Your Environment details:

* node.js version: 10.13.0                    
* browser name and version: Version 77.0.3865.120 (Official Build) (64-bit) 
* platform and version: macOS 10.15 (19A583) 
* other: Using TestCafé 1.6.0

**Comment by AlexKamaev** [comment]
I was not able to reproduce the issue. I got the error message (as far as I understand this message is injected in the page by some js code), but I did not get any error in the console.
Please see the screenshot (it's made in Windows but it is the same under Mac):
![image](https://user-images.githubusercontent.com/1678902/67196279-2dae7d00-f403-11e9-9374-e8a87cce1677.png)
Am I missing any essential steps? Could you please describe how to reproduce the issue in greater detail?

**Comment by mansn** [comment]
Ah.. now it makes sense. I found the issue, and it's originating from the Grammarly Chrome Extension I've got installed. I'm sorry for not looking more closely into the stack trace from the error in the console.

![image](https://user-images.githubusercontent.com/4518977/67242074-0bdde600-f455-11e9-8053-1c6025a15d3c.png)

Disabling Grammarly fixes it for me. This is actually not the first time I've encountered issues related to Grammarly.

I don't expect you to spend time into trying to fix this (if it's even possible). I wouldn't do it anyway, since it's such an edge case. So I'll close the issue. Thanks for looking into this.
